tests/README.md

```diff
@@ -1,18 +1,91 @@
-# Tests
+# BigchainDB Server Tests
 
-## Test Structure
+## The bigchaindb/tests/ Folder
```

Comment:
I think

> ## The `tests/` Folder

might sound better, since you're already in the `bigchaindb/` repo (they might be confused that it means the *actual* `bigchaindb/` folder).